api/v1alpha1/contour_types.go

```diff
@@ -66,16 +66,6 @@ type ContourSpec struct {
 	// +kubebuilder:validation:Minimum=0
 	Replicas int32 `json:"replicas,omitempty"`
 
-	// Namespace defines the schema of a Contour namespace. See each field for
-	// additional details. Namespace name should be the same namespace as the
-	// Gateway when GatewayClassRef is set.
-	//
-	// TODO [danehans]: Ignore Namespace when GatewayClassRef is set.
-	// xref: https://github.com/projectcontour/contour-operator/issues/212
-	//
-	// +kubebuilder:default={name: "projectcontour", removeOnDeletion: false}
-	Namespace NamespaceSpec `json:"namespace,omitempty"`
-
```

Comment:
I believe that, if we're following upstream versioning standards, removing this field requires an API bump to `v1alpha2`:

> (from https://kubernetes.io/docs/reference/using-api/deprecation-policy/ -- Rule 1) Once an API element has been added to an API group at a particular version, it can not be removed from that version or have its behavior significantly changed, regardless of track.

Reply:
@skriss thanks for pointing this out. I marked this PR as a draft and created https://github.com/projectcontour/contour-operator/issues/386 to discuss the API bump details.